### ISSUE TITLE
Move Search google functionality on slaughter to dedicated button

### ISSUE THREAD

**Issue by Snowdrama** [ludumdare/ludumdare#1794]
Suggestion to move the search to a dedicated button(maybe a magnifying glass on the side or a "more info about this theme" button or something) so that when trying to highlight the text(to copy and paste it in a discussion for example), it doesn't search.

**Comment by gonutz** [comment]
I support that. I only found out by accident that this functionality even exists. There is currently no hint at that.

**Comment by local-minimum** [comment]
I think the feature is sort of a carry over from the old site where the theme was a regular link that did the searching and now it doesn't work as well with the new style. I don't think there's a strong case for helping people to search, if you can select something it's typically very easy to make a search from the selection.

**Comment by Snowdrama** [comment]
> 
> 
> I think the feature is sort of a carry over from the old site where the theme was a regular link that did the searching and now it doesn't work as well with the new style. I don't think there's a strong case for helping people to search, if you can select something it's typically very easy to make a search from the selection.

True, I definitely agree. Another point though in your favor, the ability to highlight text allows you to use more search engines beyond google(I mean who does but the option is there)

All I'm suggesting is instead of removing the feature completely(for the 3 people who might actually use it), move it to the side so it doesn't directly interfere with the ability to highlight the text.
